### PR TITLE
splits e2e app and e2e lib-ui tests by using separate config files

### LINKED ISSUE #177: Split out Playwright tests: app vs lib
Preparatory step, so it's easy to add DOM-to-database tests later in this epic.

### PR DESCRIPTION
**Pull request by jschuurk-kr**
closes #177

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
     "prettier": "prettier --ignore-unknown --write .",
     "preview": "vite preview",
     "test": "vitest",
-    "e2e": "playwright test",
-    "e2eui": "playwright test --ui",
+    "e2e:lib-ui": "playwright test -c playwright.lib-ui.config.ts",
+    "e2e:app": "playwright test -c playwright.app.config.ts",
```
Comment (praseodym) [inline frontend/package.json:19]: Don't forget to change this in the `README.md`.

Comment (jschuurk-kr) [reply frontend/package.json:19]: thanks! done

**Review by praseodym — APPROVED** [review]
Because the config files are TypeScript code you could extract the common config from the two files instead of duplicating it, but this is good enough.

**Comment by jschuurk-kr** [comment]
> Because the config files are TypeScript code you could extract the common config from the two files instead of duplicating it, but this is good enough.

We'll have three files soon enough. :-D

My reasoning for not extracting the common parts is because the files are short enough and I anticipate/hope they'll diverge more in the near future. Happy to revisit this once I add the third file for the DOM-to-database tests.

**Review by Lionqueen94 — APPROVED** [review]

**Pull request merged**